RaccoonPG.py

```
@@ -0,0 +1,389 @@
+        super().__init__(LVL, name, HP, MP, ATTK, DEF, MATTK, MDEF, DODGE, SPD, EXP, EXPcap, POWER, CRIT, turn, inCombat, money)
+        self.POWER = 35
+        self.LVL = 1
+
+    def join_combat(self):
+        self.inCombat = True
+
+    def increase_health(self, value):
+        self.HP += value
+
+    def reduce_health(self, enemy):
+        damage = Animal.damage_formula(self, enemy, POWER, LVL, ATTK, CRIT)
```

Ruff (0.14.0)

Undefined name `POWER`

(F821)